### ISSUE TITLE
Blank white screen when I press on single password

### ISSUE THREAD

**Issue by shyaminayesh** [daper/nextcloud-passwords-app#34]
When I click on a single password to get more details about that app is giving me a blank white screen.

Expected:
Load more details about that specific password

**Comment by Larx** [comment]
I have the same issue, but only for one password. I can copy username and password, but not view and edit. No problem in the web ui.

I created this password entry in the android app, then shared it from the web ui. From this point on the problem occured (both in my original account and in the account I shared it to).

**Comment by AllirionX** [comment]
I have the same problem with all my passwords.

**Comment by chriss-de** [comment]
Here too

Is there an option to enable debug information so you can figure out why this is happening?

**Comment by jmsimonetc** [comment]
(google translate ...)
The same happens to me,
But I've noticed that it only happens to me with the passwords that I imported from the Keepass CSV.
If I generate it from the nextcloud website or in the mobile app itself, it works without problems.
I have also noticed that those generated on the web or in the mobile app are not sorted next to the ones imported in the list, but instead appear at the end.
Is there any way to get a debug log to help?
Thank you,
